packages/api-rest/src/utils/iamAuthApplicable.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { HttpRequest } from '@aws-amplify/core/internals/aws-client-utils';
+
+interface SigningServiceInfo {
+	service?: string;
+	region?: string;
+}
+
+export const iamAuthApplicableForGraphQL = (
```

Comment:
nit: I think the convention is the codebase is making the kind of function name starts with `is`, 

```suggestion
export const isIamAuthApplicableForGraphQL = (
```

Reply:
i thought about this. and chose to retain the existing naming. since the variable it is assigned to was `isIamAuthApplicable` just appended `ForGraphql` as prefix.